File: flight_checkin_links/v2/get/node/flight_checkin_links.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// What is the URL to my online check-in?
amadeus.referenceData.urls.checkinLinks.get({
    airlineCode: 'BA'
  })
  .then(function (response) {
    console.log(response);
  }).catch(function (response) {
    console.error(response);
  });

File: flight_checkin_links/v2/get/python/flight_checkin_links.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    What is the URL to my online check-in?
    '''
    response = amadeus.reference_data.urls.checkin_links.get(airlineCode='BA')
    print(response.data)
except ResponseError as error:
    raise error
